.buildkite/commands/installable-build.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/bash -eu
 
+echo "--- :github: Add GitHub to Known Hosts"
+# to allow pointing gems to git repos and branches, e.g. directly while working on release-toolkit changes
+add_host_to_ssh_known_hosts github.com
```

Comment:
EDIT: This doesn't seems to be enough to solve the issue to allow `gem 'xxx', git: 'git@github.com:…', …` references in the `fastlane/Pluginfile` to work, as it then fails on rejected `public_key` authentication.

So for now the solution to allow the `Pluginfile` to temporarily point to a branch of the toolkit ended up being to use a `https://github.com/` URL instead of a `git@github.com:` one.